### DISCUSSION TITLE
Sending repeated emails

### DISCUSSION THREAD

**Discussion by sistemasnow** [YetiForceCompany/YetiForceCRM#15807] [General]
Hello,

I have a defined workflow in helpdesk module that sends a predefined pdf daily by email to two recipients.

It works correctly except that sometimes it sends the same email several times. He should send only one but there are days that he sends two equal emails and has even sent 4.

¿How can I solve it?

(Yetiforce 6.1 in docker)

**Comment by mariuszkrzaczkowski** [comment]
Update to the latest version, was fixed problem with calling cron jobs sometimes duplicating jobs

**Reply by sistemasnow** [reply]
Hi M.

We have updated to v6.3 and we have the same problem. Some suggestion??

**Reply by sistemasnow** [reply]
Forget everything. We had an incorrect crontab configuration. We have put the line below and done it!! Thanks for everything.

*/2 * * * * www-data /var/www/html/cron/cron.sh > /var/www/html/cache/logs/cron.log 2>&1
